apps/ewallet_db/lib/ewallet_db/audit.ex

```diff
@@ -61,6 +65,10 @@ defmodule EWalletDB.Audit do
     Application.get_env(:ewallet_db, :schemas_to_audit_types)[schema]
   end
 
+  def all_for_target(record) do
+    Repo.all(Audit, target_type: get_type(record.__struct__), target_uuid: record.uuid)
```

Comment:
I just realized this. I think the 2nd and 3rd params have no effect: https://hexdocs.pm/ecto/Ecto.Repo.html#c:all/2

Same goes to the function below.